ddsrouter_test/compose/scripts/execute_and_validate_publisher.py

```diff
@@ -0,0 +1,176 @@
+# Copyright 2022 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import re
+
+import log
+import validation
+
+DESCRIPTION = """Validate that the router's publisher gets deleted"""
```

Comment:
Keep it general, as in the subscriber.

Reply:
BONUS: it would be good if you added the new disconnects validation argument to the subscriber, it will probably be useful in the future.